services/api/src/resources/task/resolvers.ts

```diff
@@ -345,7 +345,7 @@ export const taskDrushCacheClear: ResolverFn = async (
 
   const command =
     'drupal_version=$(drush status drupal-version --format=list) && \
-  if [ ${drupal_version%.*.*} == "8" ]; then \
+  if [ ${drupal_version%.*.*} == "8" ] || [ ${drupal_version%.*.*} == "9" ]; then \
```

Comment:
Is it possible in bash to do a `>= 8`?

```suggestion
  if [ "${drupal_version%.*.*}" -ge "8" ]; then \
```
```suggestion
  if [ ${drupal_version%.*.*} == "8" ] || [ ${drupal_version%.*.*} == "9" ]; then \
```

Reply:
yes and no - because the first parser matches D7 too - it pulls 7.59 (b/c no semver) and then can't do an integer comparison :cry: and throws an error (although it does then clear)
```
www-data@64846eb8a028:/app$ drupal_version=$(drush status drupal-version --format=list) && \
>   if [ ${drupal_version%.*.*} -ge "8" ]; then \
>     drush cr; \
>   elif [ ${drupal_version%.*} == "7" ]; then \
>     drush cc all; \
>   else \
>     echo "could not clear cache for found Drupal Version ${drupal_version}"; \
>   fi
bash: [: 7.59: integer expression expected
'all' cache was cleared.  
```

Reply:
(and whilst it does return 0 eventually, I'm not sure how the current task system might handle the bash error)

Reply:
bugged me just enough to switch the check logic.  At least it's now Drupal 10 safe :relieved:

Reply:
+1 thanks Toby.